README.md

```diff
@@ -200,6 +198,7 @@ maintainers = [
 contributors = [
     {family-names = "Doe", given-names= "John", email = "test@test.test", orcid = "https://orcid.org/0000-0001-2345-5678", contribution = "The main author, maintainer and tester.", contribution_begin = "2023-03-01", contribution_type = "code"},
     {family-names = "Dow", given-names= "John", email = "test2@test.test", orcid = "https://orcid.org/0000-0012-3456-7890", contribution = "Reviewer", contribution_begin = "2023-03-01", contribution_type = "review"}
+    ]
```

Comment:
there is already a ] in here

Reply:
VS Code said it was not closed and there was no matching paren highlighted :man_shrugging:

Reply:
Okay no idea which brackets were wrong but I apparently fixed it now correctly :D